crates/biome_html_analyze/tests/specs/a11y/useSemanticElements/html/valid.html

```
@@ -0,0 +1,40 @@
+<!-- should not generate diagnostics -->
+<div></div>
```

HTMLHint (1.9.2)

[error] Doctype must be declared before any non-comment content.

(doctype-first)